src/styles/blocks/catalog-grid.scss

```diff
@@ -0,0 +1,18 @@
+
+.main {
+  margin: 0;
+  justify-content: center;
+  align-items: center;
+  padding-top: 50px;
+  padding-bottom: 72px;
+  min-height: 100vh;
+  display: grid;
+  grid-template-columns: $column $column $column $column;
+  grid-template-rows: $row $row;
```

Comment:
please use [repeat ](https://developer.mozilla.org/en-US/docs/Web/CSS/repeat)function for that